catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrMetacardClientImpl.java

```diff
@@ -284,7 +304,7 @@ private boolean userSpellcheckIsOn(QueryRequest request) {
   private boolean solrSpellcheckHasResults(QueryResponse solrResponse) {
     return solrResponse.getSpellCheckResponse() != null
         && solrResponse.getResults().size() == 0
-        && solrResponse.getSpellCheckResponse().getCollatedResults().size() != 0;
+        && solrResponse.getSpellCheckResponse().getCollatedResults().size() > 0;
```

Comment:
✏️  You can also use CollectionUtils.isNotEmpty. It will null-check and size-check.